.kitchen.yml

```diff
@@ -1,96 +1,76 @@
 driver:
   name: vagrant
+  network:
+    - ["forwarded_port", {guest: 80, host: 8080}]
 
 provisioner:
   name: chef_zero
+  require_chef_omnibus: <%= ENV['CHEF_VERSION'] || 'latest' %>
```

Comment:
https://github.com/sous-chefs/haproxy/blob/master/.kitchen.yml

^here's the haproxy config that @rshade mostly wrote.